stable/sematext-docker-agent/README.md

```diff
@@ -0,0 +1,66 @@
+# Sematext Docker Agent
+
+[Sematext](https://sematext.com/) Agent for Docker collects Metrics, Events, and Logs from the Docker API for SPM Docker Monitoring & Logsene / Hosted ELK Log Management.
+
+## Introduction
+
+This chart installs the Sematext Docker Agent to all nodes in your cluster via a `DaemonSet` resource.
+
+## Prerequisites
+
+- Kubernetes 1.2+
```

Comment:
This per-requisite  is actually Kubernetes 1.8+.

For some background, [here is a doc with details on the 1.8 and 1.9 workload API changes](https://kubernetes.io/docs/reference/workloads-18-19/) and here is the [API deprecation policy](https://kubernetes.io/docs/reference/deprecation-policy/). The deprecation policy has examples for releases and when beta and alpha APIs are dropped to help you see how that works.

The `apps/v1beta1` API objects can be removed in [1.10 which is scheduled for a March 21st release](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.10/release-1.10.md)

Reply:
I understand the issue, but as we want to support older deployments I made it configurable. So default is `extensions/v1beta1`, but I it can be changed in `values.yaml` where I added a comment. The thing is, if we make it `apps/v1beta2` as default, it is bad because the majority of k8s deployments is still < 1.8. I hope this makes sense.

Reply:
There's another way to automatically get support across versions. You can use if statements in the files...

```
{{- if and (eq .Capabilities.KubeVersion.Major "1") (gt .Capabilities.KubeVersion.Minor "5") }}
```

The template language has the ability to do if/else statements. The version of kubenetes the chart will be deployed into is available in the template.

What do you think of making the version handling automatic instead of specified?

Reply:
Yeah, that is even better. Good idea. I used `{{- if .Capabilities.APIVersions.Has "apps/v1beta2" }}` to determinate if `apps/v1beta2` is available and if not, I just set api version to `extensions/v1beta1`.